### PR TITLE
Release Link FX chains of 2 different tracks (GUI) v1.0

### PR DESCRIPTION
**Pull request by mattia-marini**

### CODE REVIEW

**Comment by cfillion** [comment]
1) The 'ReaLink' name might be confused with 'ReaLinks' which already exists in the REAPER community: https://www.realinks.net/
2) Action names should use sentence case: 'Marini_ReaLink_Ui_Toggle.lua' --> 'Marini_ReaLink UI toggle.lua'
3) `GetActionCommandIDByFilename` is not necessary: ReaPack installs actions to a deterministic path so their command ID can be relied on (script command IDs are a SHA1 hash of their path)
4) Was limiting this package to be macOS-only on purpose? I see that some code checks for Windows with `reaper.GetOS()`.

**Comment by mattia-marini** [comment]
First of all thank you for the speedy feedback! 
1. After getting fixed what you listed, should I push to the fork or should I re-upload everything to [https://reapack.com/upload/reascript](https://reapack.com/upload/reascript)? 

2. I didn't find any api function to generate the SHA1, am I missing something or should I implement a function on my own?

3. I limited it intentionally since I'm not sure the UI would scale properly and I have no way of testing it on Windows. I will for sure add the support in the near future

**Comment by cfillion** [comment]
1. It's fine either way
2.  Install the script at the same location as ReaPack, then Action List > Copy selected action command ID

    For example, this opens the ReaImGui demo script on all systems:
    ```lua
    reaper.Main_OnCommand(reaper.NamedCommandLookup('_RS6b4644d86854e10895485f184942fb69ecc26177'), 0)
    ```
4. OK!

I've updated reapack.com to show the predicted command ID:
Update

**Comment by mattia-marini** [comment]
Sorry for the delay in response, it's been rough weeks... Anyway, I've pushed the recommended changes to the fork, I hope I haven't made any newbie mistakes, that's my first time contributing to open source. Thanks for your patience ;)

File: FX/Marini_FX linker/Marini_FX linker toggle autostart.lua [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+local function removeTrailingNewlines(inputString)
+    local cleanedString = inputString:gsub("[\r\n]+$", "")
+    return cleanedString
+end
+
+local function addAutoStartup()
+  local write = io.open(startupFile, "w")
+  if not write then return end
+  if content:match(pattern) then
+    print("c'è già autostart")
```
Comment (cfillion) [inline FX/Marini_FX linker/Marini_FX linker toggle autostart.lua]: Stray debug statement?

Comment (mattia-marini) [reply FX/Marini_FX linker/Marini_FX linker toggle autostart.lua]: Yeah, my bad. Removed is in [a4a2e09](https://github.com/ReaTeam/ReaScripts/pull/1301/commits/a4a2e09a5d9834a08b3174494fd620f2599e5c52)

**Pull request closed without merging**